sites/lab/hooks/use-design.mjs

```diff
@@ -116,6 +117,7 @@ const designs = {
   walburga,
   waralee,
   yuri,
+  lily,
```

Comment:
I assume these should be ordered alphabetically (same with imports above)

Reply:
This file is auto-generated and should not have been committed.